src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -170,6 +173,38 @@ int IVsEditorFactory.Close()
             return VSConstants.S_OK;
         }
 
+        private static string GetWinFormsLoaderName(IVsHierarchy vsHierarchy)
+        {
+            const string LoaderName = "Microsoft.VisualStudio.Design.Serialization.CodeDom.VSCodeDomDesignerLoader";
+            const string NewLoaderName = "Microsoft.VisualStudio.Design.Core.Serialization.CodeDom.VSCodeDomDesignerLoader";
```

Comment:
`new` is not descriptive, maybe `remote`?

Reply:
That's reasonable. However, from the perspective of this method, I'm not sure `remote` is all that descriptive either.

Reply:
I'm going to let this particular piece of feedback go for now. The comments in this method make it clear how `NewLoaderName` is used.

#wontfix